modules/veteran/app/models/veteran/flagged_veteran_representative_contact_data.rb

```diff
@@ -4,9 +4,17 @@ module Veteran
   class FlaggedVeteranRepresentativeContactData < ApplicationRecord
     self.table_name = 'flagged_veteran_representative_contact_data'
 
+    before_save :set_flagged_value_updated_at
```

Comment:
The field should be nil by default when the flag is created as long as the params used at creation don't include this value